Resources/Textures/_Umbra/Mobs/Customization/Mantidae/mantidae.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Sprites drawn necroa_",
+    "copyright": "antennae_ant, antennae_default, l_default, r_default, l_empty and r_empty drawn by necroa_, l_curved, r_curved, l_short, r_short, l_spiked, r_spiked, l_inverted and r_inverted drawn by Boaz1111 (github) based on l_default and r_default, antennae_dual and antennae_curved drawn by Boaz1111 (github) based on antennae_default.",
```

```suggestion
    "copyright": "antennae_ant, antennae_default, l_default, r_default, l_empty and r_empty drawn by necroa_ (discord), l_curved, r_curved, l_short, r_short, l_spiked, r_spiked, l_inverted and r_inverted drawn by Boaz1111 (github) based on l_default and r_default, antennae_dual and antennae_curved drawn by Boaz1111 (github) based on antennae_default.",
```

Not your problem to fix, but if you want to include this it would be nice. Not blocking if you don't want to.

---

fixed